content/services/project-consulting.mdx

```diff
@@ -0,0 +1,145 @@
+
+  ### Deliverables
+
+  We provide detailed documentation for handoff to your team as well as a maintenance plan to ensure long-term
+  sustainability for the project.
+
+</ContentSection>
+
+<ContentSection title="Funding Collaborations">
+    Our collaborations are primarily supported by grant-funded research projects. We are available to discuss your
+    ideas no matter where you are in the project lifecycle, and we are adept at scaling solutions to meet your specific
+    requirements. We could also help strengthen proposals as well. See our facilities statement.
+
+    **Partial Funding / Funding Match:** We have a limited budget for partial funding depending on need  
+    **Full Collaborator Funding:** Effort is fully paid by collaborator
+
+    <ButtonGroup>
+        <Button href="https://docs.google.com/document/d/1anrhAU6iQxAo-KzNKeiDgLd-ZiC3UsuoRAZUKOBRLB4/edit?usp=sharing" variant="primary_filled">
+            Facilities Statement
+        </Button>
+        <Button href="https://docs.google.com/document/d/1GFG9NYSh0pAQuI1n4Hba1-jC_-2ySQ6uzXltc_khnig/edit?usp=sharing" variant="primary_filled">
+            CCV Support for Grant Proposals
+        </Button>
+    </ButtonGroup>
+</ContentSection>
+
+<ContentSection title="Project Cost Estimation">
+    We try to tailor our recommendations to fit within the project’s requirements, timeline, and funding. The project cost estimations below provide a rough idea of the time, effort, and cost involved for various project sizes. Feel free to chat with us at any stage of your project; we are open to discussing alternative pay structures or finding a solution that works for all parties. 
+    <CardGroup>
+        <CostEstimateCard 
+          title="Extra Small" 
+          time="1-3" 
+          units="months" 
+          engineers="1" 
+          fte="40%"
+        />
+        <CostEstimateCard 
+          title="Small" 
+          time="6-8" 
+          units="months" 
+          engineers="1-2" 
+          fte="40%"
+        />
+        <CostEstimateCard 
+          title="Medium" 
+          time="1" 
+          units="year" 
+          engineers="1-2" 
+          fte="40%"
+        />
+        <CostEstimateCard 
+          title="Large" 
+          time="2+" 
+          units="years" 
+          engineers="2+" 
+          fte="40%"
+        />
+        \* Effort is measured in full-time equivalent (FTE) cost. We follow Brown University's
+        <a href="https://hr.brown.edu/working-brown/compensation/salary-ranges" target="_blank" rel="noopener noreferrer" >salary range guidelines</a>
+        to calculate cost.
+    </CardGroup>
+
+</ContentSection>
+
+<ContentSection title="Project Estimation Examples">
+    We provide services with limited resources at no cost to all members affiliated with Brown. For advanced computing
+    that requires extra resources, we charge a monthly fee.
```

Comment:
This paragraph doesn't work for this section? Let's take it out. This seems like leftover sentences from the rates page

```suggestion
<ContentSection title="Project Estimation Examples">
```